ajax/libs/unveil2/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "unveil2",
+  "filename": "jquery.unveil2.min.js",
+  "description": "A very lightweight jQuery plugin to lazy load images",
+  "homepage": "https://nabble.github.io/unveil2/",
+  "author": "nabble",
+  "keywords": [
+    "lazyload",
+    "jquery"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/nabble/unveil2.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nabble/unveil2.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.unveil2*.js"
```

Comment:
Let's use `**/*` here. Normally, a CDN needs all files under `dist` folder because it's release distribution. `**/*` can help cdnjs automatically fetch all files recursively under basePath.